$NetBSD: patch-configure,v 1.1 2026/08/05 12:20:03 gdt Exp $

When ginstall is found (because coreutils is installed) and used, man
pages end up installed with executable bits set.

https://github.com/coturn/coturn/issues/2037

--- configure.orig	2026-08-05 12:03:05.112985341 +0000
+++ configure
@@ -450,7 +450,7 @@ fi
 # Install shell commands
 ###########################
 
-type ginstall 2>>/dev/null
+type ginstall-avoid-because-man-pages-executable 2>>/dev/null
 ER=$?
 if [ ${ER} -eq 0 ] ; then
 	INSTALL_PROGRAM="ginstall"
